src/Security/src/Authentication.CloudFoundryCore/CloudFoundryOpenIdConnectOptions.cs

```diff
@@ -16,7 +16,6 @@ public class CloudFoundryOpenIdConnectOptions : OpenIdConnectOptions
         public CloudFoundryOpenIdConnectOptions()
         {
             AuthenticationMethod = OpenIdConnectRedirectBehavior.RedirectGet;
-            Authority = "https://" + CloudFoundryDefaults.OAuthServiceUrl;
```

Comment:
Should we just leave this file alone? I don't know that there's any advantage to making changes here since its just the one property and it is a straight assignment rather than calculation